Makefile

```diff
@@ -23,31 +23,57 @@ lint:
 	@ echo "$(BUILD_PRINT)Linting the code"
 	@ flake8 || true
 
+#-----------------------------------------------------------------------------
+# Development environment
+#-----------------------------------------------------------------------------
+
+build-dev:
+	@ echo -e '$(BUILD_PRINT)Building the dev container'
+	@ docker-compose --file docker-compose.yml --env-file .env.dev build
+
+run-dev:
+	@ echo -e '$(BUILD_PRINT)Starting the dev services'
+	@ docker-compose --file docker-compose.yml --env-file .env.dev up -d
+
+stop-dev:
+	@ echo -e '$(BUILD_PRINT)Stopping the dev services'
+	@ docker-compose --file docker-compose.yml --env-file .env.dev down
+
+#-----------------------------------------------------------------------------
+# Production environment
+#-----------------------------------------------------------------------------
+
+build-prod:
+	@ echo -e '$(BUILD_PRINT)Building the prod container'
+	@ docker-compose --file docker-compose.prod.yml --env-file .env.prod build
+
+run-prod:
+	@ echo -e '$(BUILD_PRINT)Starting the prod services'
+	@ docker-compose --file docker-compose.prod.yml --env-file .env.prod up -d
+
+stop-prod:
+	@ echo -e '$(BUILD_PRINT)Stopping the prod services'
+	@ docker-compose --file docker-compose.prod.yml --env-file .env.prod down
+
 #-----------------------------------------------------------------------------
 # Fuseki related commands
 #-----------------------------------------------------------------------------
 
-start-fuseki:
+run-fuseki:
 	@ echo "$(BUILD_PRINT)Starting Fuseki on port $(if $(FUSEKI_PORT),$(FUSEKI_PORT),'default port')"
-	@ docker-compose --file docker-compose.yml --env-file .env-dev up -d fuseki
+	@ docker-compose --file docker-compose.yml --env-file .env.dev up -d fuseki
 
 stop-fuseki:
 	@ echo "$(BUILD_PRINT)Stopping Fuseki"
-	@ docker-compose --file docker-compose.yml --env-file .env-dev down
+	@ docker-compose --file docker-compose.yml --env-file .env.dev down
 
 fuseki-create-test-dbs:
 	@ echo "$(BUILD_PRINT)Building dummy "subdiv" and "abc" datasets at http://localhost:$(if $(FUSEKI_PORT),$(FUSEKI_PORT),unknown port)/$$/datasets"
 	@ sleep 5
 	@ curl --anyauth --user 'admin:admin' -d 'dbType=mem&dbName=subdiv'  'http://localhost:$(FUSEKI_PORT)/$$/datasets'
 	@ curl --anyauth --user 'admin:admin' -d 'dbType=mem&dbName=abc'  'http://localhost:$(FUSEKI_PORT)/$$/datasets'
 
-clean-data:
-	@ echo "$(BUILD_PRINT)Deleting the $(DATA_FOLDER)"
-	@ sudo rm -rf $(DATA_FOLDER)
-
-start-service: start-fuseki fuseki-create-test-dbs
-
-stop-service: stop-fuseki clean-data
+run-fuseki-dirty: run-fuseki fuseki-create-test-dbs
```

Comment:
ditry == creating the dummy test DBs, right? 
Do we still need the test DBs, or our implementation takes care of the missing DB at the diff lading time (i.el creating the DB before the load)?